just-install.json

```diff
@@ -741,7 +741,7 @@
     "gimp": {
       "installer": {
         "kind": "innosetup",
-        "x86": "https://download.gimp.org/mirror/pub/gimp/v2.10/windows/gimp-2.10.2-setup.exe"
+        "x86": "https://download.gimp.org/mirror/pub/gimp/v2.10/windows/gimp-{{.version}}-setup.exe"
```

Comment:
The reason why the version was hardcoded is because jiup-go automatically updates it. I'm planning to fix that in jiup-go soon though, but for now, it will be automatically reverted.

Reply:
Hardcoded isn't the problem, the bot seems to update the "version" entry but not the version in the download link.
In fact, gimp is actually marked as v2.10.4 but the link redirect to 2.10.2.
I don't know how the bot work so I can't fix it myself.

Reply:
@eveldee0680 Good catch, I missed that. That is probably an issue with the download link extractor. I'll fix it now.